.vscode/tasks.json

```diff
@@ -4,7 +4,7 @@
     {
       "label": "Run Supervisor",
       "type": "shell",
-      "command": "supervisor_run",
+      "command": "sudo supervisor_run",
```

Comment:
All tasks need root, not just this one.

Reply:
Why? That is kinda my point, I'd like them to not as root so the permissions stay proper on my host.

All of them seem to run fine on my system, except `Run Supervisor CLI`, that one needs root since we don't allow Docker access without root, it seems.

Reply:
That one and frontend needs it, the others I did not see

Reply:
Right, that one starts the Docker daemon and container as well. It does other stuff too though :sweat: 

That might be not as straight forward as I thought.